packages/algorithm/rollup.config.js

```diff
@@ -1,40 +1,6 @@
-import nodeResolve from '@rollup/plugin-node-resolve';
-import sourcemaps from 'rollup-plugin-sourcemaps';
-import postcss from 'rollup-plugin-postcss';
+import { createRollupConfig } from '../../rollup.src.config';
 
 const pkg = require('./package.json');
+const config = createRollupConfig(pkg);
 
-const globals = id =>
-  id.indexOf('@lumino/') === 0 ? id.replace('@lumino/', 'lumino_') : id;
-
-export default [
-  {
-    input: 'lib/index',
-    external: id => pkg.dependencies && !!pkg.dependencies[id],
-    output: [
-      {
-        file: pkg.main,
-        globals,
-        format: 'umd',
-        sourcemap: true,
-        name: pkg.name
-      },
-      {
-        file: pkg.module + '.js',
-        format: 'es',
-        sourcemap: true,
-        name: pkg.name
-      }
-    ],
-    plugins: [
-      nodeResolve({
-        preferBuiltins: true
-      }),
-      sourcemaps(),
-      postcss({
-        extensions: ['.css'],
-        minimize: true
-      })
-    ]
-  }
-];
+export default config;
```

Comment:
By the way, I cross-checked this by comparing the results of from building with the old file versus building with the new file, as you can see from my console log below:

```
% cd packages/algorithm
% diff dist/index.es6.js prev_dist/index.es6.js                                                                                                                                                                                                                        rollup
% diff dist/index.es6.js.map prev_dist/index.es6.js.map                                                                                                                                                                                                                rollup
% diff dist/index.js prev_dist/index.js                                                                                                                                                                                                                                rollup
% diff dist/index.js.map prev_dist/index.js.map     
```